packages/eslint-plugin-patternfly-react/lib/rules/no-anonymous-functions.js

```diff
@@ -0,0 +1,82 @@
+// https://astexplorer.net/#/gist/60c119aa79dab3a3e438cc762d21490f/e2cdd0547688a64eff1d8964156cfd8fce55d21d
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        "Don't use anonmous functions without a displayName so Enzyme and React devtools show component name",
```

Comment:
```suggestion
        "Don't use arrow functions or classes without a displayName so Enzyme and React devtools show component name",
```

Reply:
Awesome, our docs now also show the component name in react dev tools, and the snapshots as well!